packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -232,9 +237,11 @@ const createEntityPageListItems = (
            </a>
         </td>
         <td>
-          <a href="${formatContentLink(uid)}">
-            ${entityId}
-          </a>
+          ${
+            universe === "false"
```

Comment:
!!accountId ? ......

https://dev.to/sanchithasr/what-is-the-double-bang-operator-in-javascript-4i3h